src/services/mail-service.js

```diff
@@ -1,149 +1,106 @@
-      + `User ${ownerName} has invited you to the room ${roomName}.\n`
-      + `Please follow this link in order to confirm the invitation: ${invitationLink}`;
-
-    const englishHtml
-      = '<p>Hello!<br/><br/>'
-      + `User <b>${ownerName}</b> has invited you to the room <b>${roomName}</b>.<br/>`
-      + `Please click here in order to confirm the invitation: <a href="${invitationLink}">Join room</a>.</p>`;
-
-    const message = {
-      from: this.emailSenderAddress,
-      to: email,
-      subject: 'Sie wurden in einen Raum eingeladen / You have been invited to a room',
-      text: `${germanText}\n\n${englishText}`,
-      html: `${germanHtml}\n${englishHtml}`
-    };
+    const subject = this.translators
+      .map(t => t('mailService:roomInvitationEmail.subject', { ownerName, roomName, invitationLink }))
+      .join(' / ');
+
+    const text = this.translators
+      .map(t => t('mailService:roomInvitationEmail.text', { ownerName, roomName, invitationLink }))
+      .join('\n\n');
 
+    const html = this.translators
+      .map(t => t('mailService:roomInvitationEmail.html', { ownerName, roomName, invitationLink }))
+      .join('\n');
+
+    const message = { from: this.emailSenderAddress, to: email, subject, text, html };
     return this._sendMail(message);
   }
 
-  sendRoomDeletionNotification({ roomName, ownerName, email }) {
+  sendRoomDeletionNotificationEmail({ email, ownerName, roomName }) {
     logger.info(`Sending delete notification to ${email}`);
 
-    const germanText
-      = 'Hallo!\n\n'
-      + `Der Benutzer ${ownerName} hat den Raum ${roomName} gelöscht.\n`;
-
-    const germanHtml
-      = '<p>Hallo!<br/><br/>'
-      + `Der Benutzer <b>${ownerName}</b> hat den Raum <b>${roomName}</b> gelöscht.<br/>`;
-
-    const englishText
-      = 'Hello!\n\n'
-      + `User ${ownerName} has deleted the room ${roomName}.\n`;
+    const subject = this.translators
+      .map(t => t('mailService:roomDeletionNotificationEmail.subject', { ownerName, roomName }))
+      .join(' / ');
 
-    const englishHtml
-      = '<p>Hello!<br/><br/>'
-      + `User <b>${ownerName}</b> has deleted the room <b>${roomName}</b>.<br/>`;
+    const text = this.translators
+      .map(t => t('mailService:roomDeletionNotificationEmail.text', { ownerName, roomName }))
+      .join('\n\n');
 
-    const message = {
-      from: this.emailSenderAddress,
-      to: email,
-      subject: 'Ein Raum, in dem Sie Mitglied waren, wurde gelöscht / A room you were a member of has been deleted',
-      text: `${germanText}\n\n${englishText}`,
-      html: `${germanHtml}\n${englishHtml}`
-    };
+    const html = this.translators
+      .map(t => t('mailService:roomDeletionNotificationEmail.html', { ownerName, roomName }))
+      .join('\n');
 
+    const message = { from: this.emailSenderAddress, to: email, subject, text, html };
     return this._sendMail(message);
   }
 
+  _sanitizeText(translation) {
+    return translation.replaceAll('\\n', '\n');
```

Comment:
This seems to mitigate an error that happened earlier in the chain (converting yml to JSON seems to sneak it in). We should fix the root cause here, I think

Reply:
I'm not sure if this won't break other translation cases.